### ISSUE TITLE
MMEX Instance crashes when trying to create a new Asset

### ISSUE THREAD

**Issue by jvn20c** [moneymanagerex/moneymanagerex#6296]
# Description of the bug

Whenever I try to create a new asset the software closes itself.

![MMEX crash jvn20c](https://github.com/moneymanagerex/moneymanagerex/assets/68383968/94b27452-a025-4bb6-9e39-0062c632f694)

As seen in the GIF, when I open the software again, it normally creates the asset.

```
Money Manager Ex
• Version: 1.6.4 64-bit
• Built on: Jun 20 2023 18:27:14
• Database version: 18 (aes128cbc)
• Git commit: 6b4cc69 (2023-06-20)
MMEX is using the following support products:
• wxWidgets 3.2.1 (wxMSW 10.0)
• wxSQLite3 4.9.1 (SQLite 3.40.0)
• RapidJSON 1.1.0
• Lua 5.3.5
• libcurl/7.82.0-DEV Schannel
• gettext 0.19.8.1
• apexcharts.js
Built with:
• CMake 3.16.2
• MSBuild 15.9.21.664
• Microsoft Visual Studio 15.0
• Microsoft Visual C++ 19.16.27035.0
• Windows SDK 10.0.14393.0
Running on:
• Windows 11 (build 22621), 64-bit edition
• Portuguese (Brazil) (windows-1252)
• 1920x1080 32bit 96x96ppi
```

**Comment by renato-mmex** [comment]
https://github.com/moneymanagerex/moneymanagerex/issues/4778
https://github.com/moneymanagerex/moneymanagerex/issues/4412

**Comment by n-stein** [comment]
I fixed this in #6089 (way back in July) for the current build.
